draft-ietf-sedate-datetime-extended.md

```diff
@@ -117,8 +117,7 @@ informative:
 
 
 This document defines an extension to the timestamp format defined in
-RFC3339 for representing additional information including a time
-zone.
+RFC3339 for representing additional information including a time zone.
```

Comment:
The gratuitous changes in this PR make this and following commits to this PR hard to review.

Reply:
Yep, that's why I recommended above (excerpted here) to review one commit at a time.

> First commit (split out to make the other commit easier to review - I'd suggest reviewing each commit separately)
>
> Normalizes line lengths to 72 characters.
> Moves each sentence into its own line for easier diffs in the future.

Do you want me to revert these changes?

Reply:
Right now I'm simply cherry-picking from this PR, so I don't have a big problem with these changes.
I just wanted to explain why this PR as a whole is hard to merge, leading us to do the cherry-picking.
There is still a lot in this PR that is worth cherry-picking, so this will keep me busy for another couple of days.

Reply:
OK. Do you want me to make any further changes on this PR in response to your feedback? Or just leave it as-is and you can pull text from it into new PRs wherever you find it helpful?

Reply:
The latter.  I marked this "changes needed" just to make clear that this shouldn't be merged wholesale.

I was distracted from working on this by Rob's comment on the charter update.  
Thank you for adding your perspective!

Reply:
(And I will continue adding comments here, both so I can remember my disposition and for you and others to see why I did not pick up a change or why I changed it while picking it up.)